src/backend/lib/redis.js

```diff
@@ -8,15 +8,15 @@ const { logger } = require('../utils/logger');
 const redisUrl = process.env.REDIS_URL || 'redis://127.0.0.1:6379';
 
 // Set MOCK_REDIS=1 to mock, MOCK_REDIS= to use real redis
-const useMockRedis = process.env.MOCK_REDIS;
+const useMockRedis = process.env.MOCK_REDIS || null;
```

Comment:
Do we need all the `nulls` when env vars are used as logical values (true, false)? `undefined` is a [falsy value](https://developer.mozilla.org/en-US/docs/Glossary/Falsy).

Reply:
I was thinking that the `nulls` might ideally and eventually be replaced with sensible default values. At least, I think that having the `nulls` there might serve as a reminder that calls to `process.env` need to be handled with care.

Reply:
Agree, let's not use `null`.  This will already use `undefined` if not set, which is what we want.

Reply:
Sure; `null` default values removed: https://github.com/Seneca-CDOT/telescope/pull/550/commits/273499f61abe90b37a46842b98f996b4f4f16a62